### ISSUE TITLE
Feature request: Batch import files

### ISSUE THREAD

**Issue by matheger** [veusz/veusz#514]
I often need to import data from multiple similar csv files in a folder. It would be nice to have a GUI feature to automate this process, for example by being able to match files based on filename patterns. The resulting data could then be created with automatic filename prefix/suffix patterns. 

Obviously one can do this by scripting it in the console, but I think a batch import option like this would warrant its own full implementation in the main program.

**Comment by canvaskan** [comment]
Agree. I'm handling the problem though combining different files to a many-column file.  But it's more reasonable and convenient to have a "multiple(batch) import files" function.
